actions/listShards.js

```diff
@@ -0,0 +1,10 @@
+
+module.exports = function listShards(store, data, cb) {
+  // https://docs.aws.amazon.com/kinesis/latest/APIReference/API_ListShards.html
+  // Currently we don't support the ExclusiveStartShardId and NextToken requests.
+  store.getStream(data.StreamName, function(err, stream) {
+    if (err) return cb(err)
+
+    cb(null, {Shards: stream.Shards})
```

Comment:
Does cb() only have a side effect and no return value? Do we need to return the value here?

Reply:
According to the this place where it is called from https://goo.gl/36Xw6n, the 2 parameters for the function is error and data. Correct me if my understanding is wrong, but that will ensure that error is null and return values is data. (in JSON format).